components/edit-collective/tiers/EditTierModal.js

```diff
@@ -374,6 +374,30 @@ function FormFields({ collective, types, values }) {
           defaultMessage: 'Amount you aim to raise',
         })}
       </FieldDescription>
+      {tier?.type === TICKET && (
+        <React.Fragment>
+          <StyledInputFormikField
+            name="singleTicket"
+            label={<FormattedMessage defaultMessage="Single Ticket" />}
+            labelFontWeight="bold"
+            mt="3"
+            flexDirection={'row'}
+            justifyContent={'space-between'}
+            alignItems={'center'}
+          >
+            {({ field, form }) => (
+              <InputSwitch
```

Comment:
I used a `switch` here instead of a `checkbox` as it seems to me that that is coherent with the existing design. In these Tier pages we use switches; for example `useStandalonePage`.